aesm-client/src/imp/aesm_protobuf/mod.rs

```diff
@@ -26,11 +26,12 @@ impl AesmClient {
         #[cfg(not(target_env = "sgx"))]
         let _ = sock.set_read_timeout(req.get_timeout().map(|t| Duration::from_micros(t as _)))?;
 
-        let req_bytes = req
-            .into()
-            .write_to_bytes()
+        let mut req_bytes = vec![0u8; 4];
```

Comment:
Do you want to use `size_of::<u32>()` in place of `4` everywhere? It's ok if not.

Reply:
Yeah, I'll update that.